docker.go

```diff
@@ -410,6 +418,42 @@ func (p Plugin) Exec() error {
 		}
 	}
 
+	if p.TarPath != "" && p.Dryrun {
```

Comment:
Let's use this opp to make sure the fields are consistent for kaniko and buildx for BUILD_ONLY/DRY_RUN. Even if code is different, we should make sure the user facing env variables are consistent. We can have two variables to maintain backward compat (alias env variables)

Reply:
yeah, then we can add  NO_PUSH (as supported in kaniko for build only)

> Even if code is different, we should make sure the user facing env variables are consistent. 

also, while updating kaniko for other providers do you think we should add support for dry_run there?